### ISSUE TITLE
Notifications are not marked as Read when link is used

### ISSUE THREAD

**Issue by marxin** [openSUSE/open-build-service#10443]
When I receive an email from OBS about a notification, I would mark it as Read similarly to what Github does.

**Comment by hennevogel** [comment]
Hm, do I understand correctly: You want to be emailed a link to mark the notification as read?

**Comment by marxin** [comment]
Yes, when I click a link from an email I would like to make the notification as read.
Very same as for github:
```
Merged #586 into master.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
```

when I click the `view it on GitHub`, it's marked as read in:
https://github.com/notifications

**Comment by hennevogel** [comment]
You may but this is probably not coming any time soon. There are things on the plate that are a bit more pressing, sorry.

**Comment by marxin** [comment]
In context of https://openbuildservice.org/2022/03/07/notifications-for-watched-items/, please implement this feature.
